src/main/java/com/stripe/model/PagingIterator.java

```diff
@@ -21,7 +21,7 @@ public class PagingIterator<T extends HasId> extends ApiResource implements Iter
       final StripeCollectionInterface<T> stripeCollection, StripeResponseGetter responseGetter) {
     this.url = stripeCollection.getUrl();
 
-    this.collectionType = stripeCollection.getClass();
+    this.pageType = stripeCollection.getPageTypeToken();
```

Comment:
can we pass this as a parameter instead of exposing getter on StripeCollectionInterface?

Reply:
Done!